packages/tasks/src/library-ui-elements.ts

```diff
@@ -541,6 +541,14 @@ transcriptions = asr_model.transcribe(["file.wav"])`,
 
 const mlAgents = (model: ModelData) => [`mlagents-load-from-hf --repo-id="${model.id}" --local-dir="./downloads"`];
 
+const sentis = (model: ModelData) => [
+	`string modelName = "[Your model name here].sentis";
+	Model model = ModelLoader.Load(Application.streamingAssetsPath + "/" + modelName);
+IWorker engine = WorkerFactory.CreateWorker(BackendType.GPUCompute, model);
+// Please see provided C# file for more details
```

Comment:
Where is the C# file provided? In the model repo?

Reply:
Hi yes, we provide it as a single file in the repo.